src/shared/containers/GetOrganizations.tsx

```diff
@@ -81,7 +82,7 @@ const GetOrganizations: FunctionComponent = () => {
       window.dataLayer.push({
         identity: {
           account_type,
-          account_created_at,
+          account_created_at: convertStringToEpoch(account_created_at),
```

Comment:
Is there anything we'd have to change in the Google Optimize dashboard since the data type is changing to `number`?

Reply:
Good question but nope.